src/main/webapp/dashboard/controller/Dashboard.controller.js

```diff
@@ -2,500 +2,193 @@ sap.ui.define(
   [
     "./AbstractMasterController",
     "sap/ui/model/json/JSONModel",
-    "../resources/libs/vega5",
-    "../resources/libs/vega-embed5",
+    "sap/ui/integration/widgets/Card"
```

Comment:
This would be better to have an on the Dashboard.view.xml and then do the binding there.